morecantile/models.py

```diff
@@ -121,6 +126,10 @@ def crs(self) -> CRS:
     @classmethod
     def load(cls, name: str):
         """Load default TileMatrixSet."""
+        warnings.warn(
+            "TileMatrixSet.load will be deprecated in version 1.1.0",
+            DeprecationWarning,
+        )
```

Comment:
We don't need this anymore and it will make things cleaner IMO. Model shouldn't have to care about this.